### ISSUE TITLE
BOSSA not working in MacOS Sierra

### ISSUE THREAD

**Issue by WZ9V** [shumatech/BOSSA#53]
MacOS Sierra 10.12.5, BOSSA GUI looks like it starts but then just exits without doing anything.  this is with the supplied version 1.8 DMG from the releases.

Hardware is Early 2015 MacBook 12 using external hub to Duet Wifi

Not sure what is needed to recompile on MacOS so have not tried that option.

**Comment by odboy** [comment]
Hi WZ9Z, 
Do you already solve this problem?
It's same that after I mistakenly selected Bluetooth USB port instead J-Link USB port。

**Comment by shumatech** [comment]
I'll be doing the 1.9 release on High Sierra so hopefully that will fix things.

**Comment by shumatech** [comment]
I'm running 10.13.3 HS with no issues right now.  I'm using homebrew wxmac 3.0.3.1_1.

**Comment by rob-miller** [comment]
Not sure if the OP was trying to connect to a DuetWifi or PanelDUE, the latter being the device this software is specified for upgrading.  For the PanelDUE, the USB port is not enabled by the pre-loaded firmware.  Thus the port will not appear in the drop-down list (and nothing will work) until the 'erase' button has been pressed and the device subsequently reset.  Before I learned this, I selected the bluetooth port while trying things as odboy referenced, and got it (Bossac v1.8) into the loop described where it starts and immediately exits until the relevant files under ~/Library/ are deleted.  I confirm that v1.9 compiles easily with homebrew wxmac, but upon learning about needing to erase I went back to the mac air with Bossac 1.8 on it and was successful with that.
